src/i18n/locales/en-US.json

```diff
@@ -293,6 +293,16 @@
   "components.walletinit.walletinitscreen.restoreWalletButton": "Restore wallet",
   "components.walletinit.walletinitscreen.restoreShelleyWalletButton": "Restore wallet (Shelley Testnet)",
   "components.walletinit.walletinitscreen.title": "Add wallet",
+
```

Comment:
there shouldn't be blank lines here (may be it won't generate an error in crowdin, but let's keep consistency)